src/saveload/cheat_sl.cpp

```diff
@@ -17,8 +17,8 @@
 #include "../safeguards.h"
 
 static const SaveLoad _cheats_desc[] = {
-	SLE_VAR(Cheats, magic_bulldozer.been_used, SLE_BOOL),
-	SLE_VAR(Cheats, magic_bulldozer.value, SLE_BOOL),
+	SLE_CONDVAR(Cheats, magic_bulldozer.been_used, SLE_BOOL, SL_MIN_VERSION, SLV_REMOVE_MAGIC_BULLDOZER),
+	SLE_CONDVAR(Cheats, magic_bulldozer.value, SLE_BOOL, SL_MIN_VERSION, SLV_REMOVE_MAGIC_BULLDOZER),
```

Comment:
Maybe you should use the current value to set the settings, as otherwise games that have magic bulldozer enabled now all of a sudden no longer have. A small effort to change the setting ofc, but it might frustrate the user.

Reply:
Hmm, that's a good idea. Will see what I can do!